File: 1st semester/Agenda 05/NewInfo.py
age = int(input("Você tem quantos anos? "))
if age < 60:
    print("Vá para a fila comum!")
elif 60 <= age <= 80:
    print("Vá para a fila prioritária!")
else:
    print("Vá para a fila 80+!")

File: 1st semester/Agenda 05/NewInfo.java
package newinfo;

import javax.swing.JOptionPane;
public class NewInfo {
    public static void main(String[] args) {
        int age;
        age = Integer.parseInt(JOptionPane.showInputDialog("Quantos anos você tem?"));
        if (age < 60) {
            JOptionPane.showMessageDialog(null, "Vá para a fila comum");
        } else if (age >= 60 && age <= 80) {
            JOptionPane.showMessageDialog(null, "Vá para a fila prioritária");
        }
        else{
            JOptionPane.showMessageDialog(null, "Vá para a fila 80+");
        }
    }

}
